lib/jsonPointer.js

```diff
@@ -117,8 +117,8 @@ function concatJsonPointer(traceFromParent, targetInRoot) {
 * @param {string} version - The version we are working on
 * @returns {string} - the concatenated json pointer
 */
-function getJsonPointerRelationToRoot(refValue, traceFromKey) {
-  let targetInRoot = '/components';
+function getJsonPointerRelationToRoot(refValue, traceFromKey, version) {
+  let targetInRoot = isSwagger(version) ? '' : '/components';
```

Comment:
Can we mention comment on why is it empty for swagger? Also should it not be `/definitions`?

Reply:
For swagger 2.0 we don’t have a global container as we have for 3.0 and 3.1 for reusable structures  (components), we have definitions, securityDefinitions etc. And those are solved depending on the type of object they contain. @erfemega